src/components/Layout/index.js

```diff
@@ -0,0 +1,13 @@
+import React, { Fragment } from "react"
+import "./Layout.scss"
+
+import Toolbar from "./Toolbar/Toolbar"
```

Comment:
redenumeste fisierul in **index.js** si importeaza **import Toolbar from "./Toolbar"**